src/app/inventory-page/StoreBucketDropTarget.m.scss

```
@@ -1,5 +1,5 @@
 .canDrop {
-  background-color: rgba(200, 200, 200, 0.2);
+  background-color: rgba(255, 255, 255, 0.2);
 }
 .over {
   box-shadow: inset 0 0 6px 0 rgba(200, 200, 200, 0.7);
```

Maybe update this one too, to be consistent.
```suggestion
  box-shadow: inset 0 0 6px 0 rgba(255, 255, 255, 0.7);
```